lib/fastlane/plugin/wpmreleasetoolkit/actions/common/comment_on_pr.rb

```diff
@@ -10,7 +10,10 @@ class CommentOnPrAction < Action
       def self.run(params)
         require_relative '../../helper/github_helper'
 
-        reuse_identifier = Fastlane::Helper::GithubHelper.comment_on_pr(
+        token = params[:access_token]
+        github_helper = Fastlane::Helper::GithubHelper.new(github_token: token)
```

Comment:
Self-Review: This Action is the only one that has already a `GITHUB_TOKEN` in their ConfigItens, so do not create a breaking change with the consumers, I leave it like this, instead of using the `github_client` param as in the other actions.